host-configs/llnl/toss_4_x86_64_ib_cray/clang@18.0.0_hip.cmake

```diff
@@ -53,12 +53,10 @@ set(ENABLE_MPI ON CACHE BOOL "")
 #------------------------------------------------------------------------------
 
 #_blt_tutorial_useful_hip_variables_start
-set(_rocm_root "/opt/rocm-5.6.0")
+set(ROCM_ROOT_DIR "/opt/rocm-6.2.1" CACHE PATH "")
 
 set(ENABLE_HIP ON CACHE BOOL "")
 
-set(HIP_ROOT_DIR "${_rocm_root}/hip" CACHE STRING "")
-
 set(CMAKE_HIP_ARCHITECTURES "gfx90a" CACHE STRING "")
 
 set(CMAKE_EXE_LINKER_FLAGS "-Wl,--disable-new-dtags -L${_rocm_root}/hip/../llvm/lib -L${_rocm_root}/hip/lib -Wl,-rpath,${_rocm_root}/hip/../llvm/lib:${_rocm_root}/hip/lib -lpgmath -lflang -lflangrti -lompstub -lamdhip64  -L${_rocm_root}/hip/../lib64 -Wl,-rpath,${_rocm_root}/hip/../lib64  -L${_rocm_root}/hip/../lib -Wl,-rpath,${_rocm_root}/hip/../lib -lamd_comgr -lhsa-runtime64 " CACHE STRING "")
```

Comment:
This line still uses _rocm_root instead of ROCM_ROOT_DIR

Reply:
Good point. I am going to try remove that line first and hope it works, then will fixup :)